src/styles/styles.js

```diff
@@ -755,32 +755,32 @@ const styles = {
         borderRightWidth: 0,
     },
 
-    chatSwitcherSingleAvatar: {
+    singleAvatar: {
         height: variables.avatarSizeNormal,
         width: variables.avatarSizeNormal,
         borderRadius: variables.avatarSizeNormal,
     },
 
-    chatSwitcherBigAvatar: {
+    bigAvatar: {
```

Comment:
I don't know if I like the name `bigAvatar` - this is actually just the standard, normal size avatar we'd expect to see across the application and not necessarily "big"

Reply:
How about `normalAvatar `instead of `bigAvatar`?

Reply:
I would just switch the order to be `avatarNormal`

Reply:
avatarNormal is already declared.

Reply:
Is there any reason why we can't just reuse it then? Meaning, we don't need to write new styles, just reuse that same component that already exists.

Reply:
Ok.

Reply:
+1